### DISCUSSION TITLE
How do I import a modelfile from openwebui.com?

### DISCUSSION THREAD

**Discussion by Darin755** [open-webui/open-webui#1304] [Q&A] [answered]
I am trying to import from openwebui.com but I do not know how. When I hit Get it just asks me to create an account which seems odd.

**Comment by JuanjoJG-WATA** [comment, marked as answer]
Do you mean a modelfile?
I had the same problem, first I had to install Ollama, download llama2:latest and if the modelfile was based on it, it would work

**Reply by Darin755** [reply]
I just didn't see an way way to download the model file. I just ended up copy and pasting which isn't ideal.
